pkg/actions/commands.go

```diff
@@ -1011,6 +1011,7 @@ func Commands() {
 			Aliases: []string{"mg"},
 			Usage:   "Gathers logs and project files to aid diagnosis of Codewind errors",
 			Flags: []cli.Flag{
+				cli.StringFlag{Name: "conid", Value: "local", Usage: "Triggers mustgather collection for the `remote` codewind instance (_must_ have currently configured Kubectl connection!)", Required: false},
```

Comment:
you have conid here but in the README.md its connid

Reply:
I'll change the README to conid as that's what it is everywhere else in the code